frontend/cs/scs/builder.go

```diff
@@ -686,3 +687,57 @@ func (builder *builder) ToCanonicalVariable(v frontend.Variable) frontend.Canoni
 		return term
 	}
 }
+
+// GetWireGates returns the pairs (gateId, wireLocation) for the given wires in
```

Comment:
naming wise, we use "constraint" everywhere, not gate, I'ld prefer to keep it that way for consistency even though it means the same thing here :)

Reply:
👍  will change.

Reply:
Renamed.